### PR TITLE
Update sdk and demo for "ledgeritems" endpoints

### PR DESCRIPTION
**Pull request by Leh2**
Opdaterer SDK og Demo for endpoint `ledgeritems`

### CODE REVIEW

File: DineroClientSDK/DineroPortableClientSDK/DineroSettings.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ private DineroSettings(ClientCredentials client, DineroOauth2Client.Authorizatio
 
         public Uri GetBaseUri(string version = null)
         {
-            return new Uri(string.Format("{0}/v{1}/", GetBaseApiUri(), version ?? GetApiVersion()));
+            return new Uri($"{GetBaseApiUri()}/v{version ?? GetApiVersion()}/");
```
Comment (larsnikolajsendinero) [inline DineroClientSDK/DineroPortableClientSDK/DineroSettings.cs]: vi skal lige cleare med SpringFeed om det er ok at bruge c# 6 features...

**Comment by sirpadk** [comment]
With the risk of asking a stupid question.

I am taking yet a nother try to develop an app that can work with the Dinero API.

I would rather use the nuget package, but it seems to be the case that the nuget package ("DineroPortableClientSDK": "1.0.5681.22946") is not up to date with the latest version on the GitHub.

**Comment by larsnikolajsendinero** [comment]
The SDK nuget package will be updated in a near future. We will set it up to update automatically when anything is being pushed to the repository, but until it is ready, you will need to clone the repository

**Comment by sirpadk** [comment]
Ok, so I need to clone the repository, build it, and then include the dll in my application?

**Comment by larsnikolajsendinero** [comment]
yes, for now :-)

**Comment by sirpadk** [comment]
Hi again,

Firstof all, it is great to see the activity on the SDK. Especially the new feature that allows creating and booking of items.

I am working on a windows app (UWP) to work with Dinero. and I would like to ask some questions in regards to that:

- I've created a Unit test project, one of the tests creates a ledger item. I would like to be able to delete it after the test passed, and there is no visible way to delete a ledger item from the SDK. Am I missing something? What are options? If the option is not available, are there any plans to implement it?

- Regarding the open source part, I would like to know if you are open for contribution from ppl like me. Right now I have created a separate SDK project, where I extend som of the capabilities of your SDK, but it would be nice to know what are my options to contribute otherwise.

I hope that this is the correct channel to contact you developers of the SDK, if not please correct me.

**Comment by Leh2** [comment]
HI @sirpadk,

Deletion of ledger items will we ready after we merge https://github.com/DineroRegnskab/dinero-csharp-sdk/pull/10 (later today).

Contributions are welcome, but please take a look at http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/ to ensure the pull request is ready to merge when created.

For questions and issues use issues on github, as you did with https://github.com/DineroRegnskab/dinero-csharp-sdk/issues/9.

**Comment by larsnikolajsendinero** [comment]
#10 has been merged now

**Pull request merged**